app/assets/sass/nhs_fonts.scss

```diff
@@ -1,8 +1,20 @@
-
-
 @font-face {
   font-family: 'nta';
   src: url(/public/fonts/NeueFrutigerComRegular.woff);
   font-weight: normal;
   font-style: normal;
-}
\ No newline at end of file
+}
+
+@font-face {
+  font-family: 'Frutiger Regular';
```

Comment:
You mean 'nta' is that right? That's going to be legacy and I want to get to the point of removing it altogether. I believe once form elements including buttons have had a "design debt" exercise, we should be able to get rid of it...